src/main/resources/messages.json

```diff
@@ -7,11 +7,11 @@
       "description": "This notification most fields of the message data model populated, even though many are unused. This is so there's something in place when we start experimenting with new usages. It is set to expire in one year.",
       "descriptionShort": "This notification uses the new 'message' data model.",
       "messageType": "notification",
-      "goLiveDate": "2017-08-14T10:50",
-      "expireDate": "2018-08-14T10:50",
       "featureImageUrl": null,
       "priority": null,
-      "audienceFilter": {
+      "filter": {
+        "goLiveDate": "2017-08-14T10:50",
```

Comment:
As long as we're breaking-changes-ing the `JSON` format, maybe let's reconsider the `goLiveDate` and `expireDate` names. 

[SAML uses `NotBefore` and `NotOnOrAfter`](https://www.samltool.com/generic_sso_res.php). I like those (the concepts, not the capitalization) for clarity. (If `NotOnOrAfter` is a Date, should the message show on that date? Clearly not.) and for their *not* saying "Date", since here it might not be a Date, it might be  date-time.